pages/evaluation.md

```diff
@@ -1,32 +1,38 @@
-# Butterfly Hybrid Detection: Evaluation
-This challenge is an anomaly detection problem. The goal is to find hybrid butterfly instances. There are two sets of images considered in the test set, as described below.
+# Beetles as Sentinel Taxa: Evaluation
+This challenge is an out-of-sample prediction challenge. The goal is to predict the drought status at a location on a specific date based on images of carabid beetle specimens collected at the location on that date. This will be an out-of-sample challenge because training data will be assembled from one site of locations, and test data will assembled from a different set of locations. All locations will be NEON sites.
```

Comment:
```suggestion
This challenge is an out-of-sample prediction challenge. The goal is to predict the drought status at a location on a specific date based on images of carabid beetle specimens collected at the location on that date. This will be an out-of-sample challenge because training data will be assembled from one site of locations, and test data will be assembled from a different set of locations. All locations will be NEON sites.
```